cmd/zoekt-sourcegraph-indexserver/cleanup.go

```diff
@@ -158,8 +176,15 @@ func shardRepoNames(path string) ([]string, error) {
 		return nil, err
 	}
 
+	ts, err := zoekt.LoadTombstones(path)
```

Comment:
i don't know if this works, but should we make ReadMetadataPath exclude tombstoned repos? I suppose it would break other stuff too much, for example untombstoning.

Alternatively could we hydrate in tombstone state in ReadMetadataPath? EG add a Tombstone field to the struct and make sure it isn't json serializable...

thinking further along that line, we could make it json serializable and just rely on meta files to signal tombstones.